atlp-rwanda/tech-strikers-bn-backend
# #175636835 Setup dotenv Variables

#### What does this PR do?
It sets up environment variables for Barefoot nomad
#### Description of Task to be completed?
- Created `.env` file to hold all our environment variables
- Created `.env.example` file to show environment variables. You can change the text and add your suitable data
#### How should this be manually tested

run
- `git clone https://git.github.com/atlp-rwanda/tech-strikers-bn-backend`
- `cd tech-strikers-bn-backend`
- `git checkout ch-setup-dotenv-175636835`
- `npm install`
- create `.env` in root folder in this file create this variable
` API_VERSION = 'v1'`
- `npm run dev` and you will see in terminal v1
#### Any background context you want to provide?
All env variables are set up according to my local machine Postgres
#### What are the relevant pivotal tracker stories?
[#175636835](https://www.pivotaltracker.com/story/show/175636835)
#### Screenshots (if appropriate)
![Screenshot from 2020-11-11 12-59-48](https://user-images.githubusercontent.com/57761095/98811802-14bba400-242a-11eb-8f13-62bd36279009.png)
![Screenshot from 2020-11-11 12-59-42](https://user-images.githubusercontent.com/57761095/98811812-16856780-242a-11eb-88c8-32094c3abfff.png)
![Screenshot from 2020-11-11 12-59-31](https://user-images.githubusercontent.com/57761095/98811814-17b69480-242a-11eb-9557-416bcd65a590.png)

#### Questions:
